libs/langgraph/langgraph/types.py

```diff
@@ -417,7 +428,7 @@ def node(state: State):
                 # as part of the interrupt information.
                 \"what is your age?\"
             )
-            print(f\"> Received an input from the interrupt: {answer}\")
+            print(f"> Received an input from the interrupt: {answer}\")
```

Comment:
I believe the `\` is necessary in this case

```suggestion
            print(f"> Received an input from the interrupt: {answer}\")
```

Reply:
this got me an issue at the ci but I pushed the old version anyways